### ISSUE TITLE
include option of D3-light.js?

### ISSUE THREAD

**Issue by jread-usgs** [USGS-VIZLAB/vizlab#296]
https://github.com/datawrapper/d3-light

This is a lighter (order of magnitude smaller minified) D3 subset, which I think includes basics that we would be using on some of the lighter visualizations.

**Comment by aappling-usgs** [comment]
D3 v4 is modular, unlike v3 - maybe it would make sense to pull in specific modules rather than the particular wrapper linked above, which is "15 commits ahead, 1566 commits behind d3:master"?

**Comment by jread-usgs** [comment]
Good point, yes. If we can just get the modules we want (and they can be minified), :+1:
